lib/peep.rb

```diff
@@ -0,0 +1,53 @@
+require 'pg'
+
+class Peep
+
+  attr_reader :id, :date, :time, :message, :sender
+
+  def initialize(id:, date:, time:, message:, sender:)
+
+    @id = id
+    @date = date
+    @time = time
+    @message = message
+    @sender = sender
+
+  end
+
+  def self.all
+
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+      else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    peeps = connection.exec("SELECT * FROM peeps ORDER by date DESC, time DESC;")
+
+    peeps.map do |peep|
+      Peep.new(id: peep['id'], date: peep['date'], time: peep['time'], message: peep['message'], sender: peep['sender'])
+    end
+
+
+  end
+
+  def self.create(message:, sender:)
+
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+      else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    date = "#{Time.new.year}-#{Time.new.month}-#{Time.new.day}"
+    time = "#{Time.new.hour}:#{Time.new.min}"
```

Comment:
Might have been better to use a timestamp provided by SQL? saves a few lines of code.